apps/HelloPyTorch/src/add_generator.cpp

```diff
@@ -15,9 +15,9 @@ Func add_(const Input &input_a, const Input &input_b) {
 
 class AddGenerator : public Generator<AddGenerator> {
 public:
-    Input<Buffer<>> input_a{"input_a", 4};
-    Input<Buffer<>> input_b{"input_b", 4};
-    Output<Buffer<>> output{"output", 4};
+    Input<Buffer<void, 4>> input_a{"input_a"};
```

Comment:
This is a little unfortunate. Is it possible to make Buffer<4> work? I don't think that's doable in c++, right?

Reply:
No, definitely not. IMHO this is a minor price to pay.